custom/aaa/models.py

```diff
@@ -419,17 +419,19 @@ def agg_from_child_health_case_ucr(cls, domain, window_start, window_end):
 
         return """
         INSERT INTO "{child_tablename}" AS child (
-            domain, person_case_id, child_health_case_id, opened_on, closed_on
+            domain, person_case_id, child_health_case_id, mother_case_id, opened_on, closed_on
         ) (
             SELECT
                 %(domain)s AS domain,
-                child_health.person_case_id AS person_case_id,
-                child_health.doc_id AS child_health_case_id,
-                child_health.opened_on AS opened_on,
-                child_health.closed_on AS closed_on
+                child_health.person_case_id,
+                child_health.doc_id,
+                child_health.mother_case_id,
+                child_health.opened_on,
+                child_health.closed_on
             FROM "{child_health_cases_ucr_tablename}" child_health
         )
         ON CONFLICT (person_case_id) DO UPDATE SET
+           mother_case_id = EXCLUDED.mother_case_id,
```

Comment:
Is the reason you are only updating these two fields because they are the only ones that could have changed?

Reply:
Yeah. I think there's no reasonable worflow where the other's change. I'm not sure if a mother_case_id could get linked at a later time which is why I am updating it here